lib/domain/exceptions/exceptions.dart

```diff
@@ -0,0 +1,3 @@
+class DataRetrievingException implements Exception {}
+
+class NoInternetException implements Exception {}
```

Comment:
You throwing these exceptions but do not uses them - so looks like they are redundant

Reply:
done